src/js/entry.js

```diff
@@ -116,6 +117,16 @@ export function bootstrap(libjwt, initFunc) {
                         libjwt.jwt.logoutAllTabs();
                     });
                 },
+                getUserEntitlements: () => {
+                    const keys = Object.keys(localStorage).filter(key => (
```

Comment:
we should probably add a check here to make sure localStorage is enabled in the first place